### PR TITLE
Hotfix/validator manager

### PR DESCRIPTION
**Pull request by cowboy-bebug**
Sorry guys I pushed buggy code (from this https://github.com/plugblockchain/plug-blockchain/commit/dbcbd2a7bef857c5eeb24a1774fae64f565fb8e7). Checked with
```console
$ cargo check -p prml-validator-manager && cargo test -p prml-validator-manager
```

### First commit resolves this issue:
Screen Shot 2020-07-23 at 4 29 38 PM

### Second commit resolves this issue:
Screen Shot 2020-07-23 at 4 29 20 PM

### CODE REVIEW

**Comment by jordy25519** [comment]
Thanks, how did this pass CI before though?
Do we need to add a new CI step to support this module?

**Comment by cowboy-bebug** [comment]
Yeah let me investigate that and get back to you.

File: .circleci/config.yml [hunk 1]
```diff
@@ -60,7 +60,7 @@ commands:
   cargo-run-test:
     steps:
       - run:
-          command:
+          command: |
```
Comment (cowboy-bebug) [inline .circleci/config.yml:63]: This was the culprit. The pipeline was running `cargo --version cargo test --release --all` in a single line so `cargo test` part never executed.

Comment (cowboy-bebug) [reply .circleci/config.yml:63]: As for the doc tests, `cargo test --all` doesn't test docs and we have to run `cargo test --doc` separately.
But `cargo test -p some-crate` tests for that crate's doc (little bit confusing) <- and this is how the doc error was caught.

**Comment by cowboy-bebug** [comment]
All the tests are now fixed 😄
Also added some useful scripts (https://github.com/plugblockchain/plug-blockchain/pull/117/commits/6fb409dda42114ef13681cc79edb82da71030de3) for similar debugging tasks in future.

**Comment by jordy25519** [comment]
fixes lvgtm.
Those bash scripts are 🔥 (did you chmod+x them?)

**Review by jordy25519 — APPROVED** [review]

**Comment by cowboy-bebug** [comment]
Yep all the executables are `x` (`logger.sh` is not):
```
-rwxr-xr-x   1 eric  staff  1495 27 Jul 08:48 bootstrap
-rw-r--r--   1 eric  staff   441 27 Jul 08:49 logger.sh
-rwxr-xr-x   1 eric  staff   703 27 Jul 08:49 overwrite-ui
-rwxr-xr-x   1 eric  staff  2018 27 Jul 08:49 test-wasm
```

**Pull request merged**